src/submodules/Directory.Build.props

```diff
@@ -1,2 +1,9 @@
-<!-- Intentionally breaks the inheritance chain so submodules do not inherit build settings from this repo. -->
-<Project />
+<Project>
+  <!--
+    For everything but Microsoft.DotNet.Web.Spa.ProjectTemplates, intentionally break the inheritance chain so
+    submodules do not inherit build settings from this repo. For Microsoft.DotNet.Web.Spa.ProjectTemplates, break the
+    build if referenced file does not exist.
+  -->
+  <Import Project="..\ProjectTemplates\Directory.Build.props"
+      Condition=" '$(MSBuildProjectName)' == 'Microsoft.DotNet.Web.Spa.ProjectTemplates' " />
```

Comment:
When I update the spa-template repo to build outside the aspnetcore repo, I'll make this logic unnecessary.